src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,30 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  selectTodo: Todo;
+  setSelectTodo: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ selectTodo, setSelectTodo }) => {
+  const [loading, setLoading] = useState(true);
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(selectTodo.userId).then(person => {
+      setUser(person);
+      setLoading(false);
+    });
```

Comment:
Consider adding error handling for the `getUser` function call. If the API request fails, the component should handle the error gracefully, possibly by displaying an error message to the user.